template/src/navigation/CustomTabBar.tsx

```diff
@@ -6,7 +6,7 @@ import {StyleSheet, Text, TouchableOpacity, View} from 'react-native'
 import React from 'react'
 import {SafeAreaView} from 'react-native-safe-area-context'
 
-function CustomTabBar({routeName, navigation}) {
+function CustomTabBar({routeName, navigation}: unknown) {
```

Comment:
Does the react navigation have this props type?

Reply:
Updated